examples/job/generate-slabs-al2o3.py

```diff
@@ -0,0 +1,250 @@
+                                                                      content=content,
+                                                                      owner_id = ORGANIZATION_ID)
+
+
+# ## Create the Workflow
+# Create a workflow based on the Variable-cell Relaxation workflow available [here](https://platform.exabyte.io/analytics/workflows/NAdKjws8qieKWeYnL)
+
+# In[]:
+
+
+# Start by finding a unit cell optimization workflow on the Exabyte platform.
+# The bank workflow ID can be found in the bank workflows URL
+exabyte_bank_workflows_endpoint = BankWorkflowEndpoints(*ENDPOINT_ARGS)
+bank_workflow_id = "NAdKjws8qieKWeYnL"
+al2o3_workflow = exabyte_bank_workflows_endpoint.copy(bank_workflow_id, account_id = ORGANIZATION_ID)
+
+# Set the names to something easy to recognize, and upload
+exabyte_workflows_endpoint = WorkflowEndpoints(*ENDPOINT_ARGS)
+al2o3_workflow['name'] = "Alumina_Cell_Relax"
+al2o3_cellopt_workflow = exabyte_workflows_endpoint.create(al2o3_workflow)
+
+
+# ## Create the Job
+# Then create the jobs to optimize these unit cells
+
+# In[]:
+
+
+# Get the default projectID for the organization account (this can be substituted with a userID)
+exabyte_projects_endpoint = ProjectEndpoints(*ENDPOINT_ARGS)
+project_id = exabyte_projects_endpoint.list({"isDefault": True,
+                                             "owner._id": ORGANIZATION_ID})[0]["_id"]
+
+# Create the compute configuration for the jobs on Azure
+exabyte_jobs_endpoint = JobEndpoints(*ENDPOINT_ARGS)
+job_config = {"ppn": 16,
+              "queue": "OF",
+              "nodes": 1,
+              "time_limit": "00:20:00",
+              "cluster": "cluster-007"}
+compute = exabyte_jobs_endpoint.get_compute(**job_config)
+
+# Create the Al2O3 job
+al2o3_job = exabyte_jobs_endpoint.create_by_ids([al2o3_cell_material],
+                                                al2o3_cellopt_workflow["_id"],
+                                                project_id,
+                                                ORGANIZATION_ID,
+                                                "al2o3_cellopt",
+                                                compute)
+
+
+# Finally, submit the jobs
+
+# In[]:
+
+
+exabyte_jobs_endpoint.submit(al2o3_job[0]["_id"])
+
+
+# # Generate the Slabs
+
+# We begin by extracting the relaxed unit cell from the jobs
+
+# In[]:
+
+
+# Extract job ID
+al2o3_job_id = al2o3_job[0]["_id"]
+
+# Get a list of files
+al2o3_files = exabyte_jobs_endpoint.list_files(al2o3_job_id)
+
+# Find the CONTCAR
+for file in al2o3_files:
+    if file["name"] == "CONTCAR":
+        al2o3_file_metadata = file
+
+# Get a download URL for the CONTCAR
+al2o3_contcar_signed_url = al2o3_file_metadata['signedUrl']
+
+# Download the contcar to memory
+al2o3_response = urllib.request.urlopen(al2o3_contcar_signed_url)
+
+# Write it to disk
+with open("al2o3_relaxed.vasp", "wb") as outp:
+    outp.write(al2o3_response.read())
```

Comment:
You get URL, download and write to disk at least 3 times in these notebooks - isolate to a util fn??

Reply:
Added a util function to utils, updated the wulff and al2o3 notebooks to use it. It might make sense to leave this as-is in the list-files notebook, since the purpose there is to teach how the call is done